tests/provisioning-manifest-snapshot.yml

```diff
@@ -23,5 +23,6 @@
     - 'mvn:org.jahia.modules/default-skins'
     - 'mvn:org.jahia.modules/richtext-ckeditor5'
     - 'mvn:org.jahia.modules/jcontent'
+    - 'mvn:org.jahia.modules/test-ckeditor5-config'
```

Comment:
I see `org.foo.modules` in the pom.xml so probably need to change?

Also maybe better to change to `org.jahia.test`